releases/v0.8.0.md

```diff
@@ -0,0 +1,67 @@
+# Release Notes for v0.8.0
+
+Release Date: Oct TBD, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+* Confidential Data Hub (CDH) allows users to do in-TEE secret processing.
+  * Supports sealed Kubernetes secrets that can be decrypted by KMS or stored in KBS
+* Support added for multi-KBS model, where one KBS validates evidence using Attestation Service while another provides resource.
+* Enable pulling container images on host which enables image sharing between multiple pods.
+* Upstream containerd now shipped in this release, with exceptions of the following projects:
+  * Enclave-cc
+  * TBD
+
+## Hardware Support
+
+Confidential Containers is tested with attestation on the following platforms:
+
+* Intel TDX
+* AMD SEV(-ES)
+* Intel SGX
+
+The following platforms are untested or partially supported:
+
+* IBM Secure Execution (SE) on IBM zSystems (s390x) running LinuxONE
+* AMD SEV-SNP
+
+The following platforms are in development:
+
+* ARM CCA (TODO: Find out what is the status of this?)
```

Comment:
Remove TODO. If there is sentiment that the CCA work is no longer active then we can drop the secion.
```suggestion
* ARM CCA
```

Reply:
There is still ARM development happening, although idk what the timeilne is.

Reply:
Still see this TODO